tests/strategies/parse/test_text_csv.py

```diff
@@ -159,7 +162,7 @@ def test_csv_dialect_enum_fails() -> None:
 
     config = {
```

Comment:
This whole config should be changed to be similar to the previous test - with the intended "wrong" dialect in the "configuration" dict.